config/runtime.cfg

```diff
@@ -306,6 +306,18 @@
 #          Default:  false
 #          Example:  PHYSICAL_CANCEL  TRUE
 
+# Environment name:  COB_LOAD_GLOBAL
+#   Parameter name:  load_global
+#          Purpose:  tell the system loader to provide symbols in CALLed
+#                    programs globally, allowing symbols to be found for linked
+#                    libraries later
+#             Type:  boolean
+#             Note:  COBOL CALLs will always find symbols in already CALLed or
+#                    pre-modules; this setting is mostly an advise to the system,
+#                    not all systems are capable of loading libraries global/local
+#          Default:  false
```

Comment:
As this changes the current default - please add a NEWS entry at the appropriate place, telling about this change and the option to use the setting for the old behavior

Reply:
Is this the behavior that we want to have as default? Or should we keep the default as it was before this change and only add this option to change it when needed?

Reply:
Previously the behaviour was depending on the underlying system (is dlopen there and works, what is the default for its option and otherwise the default for lt_dlopen), with _often_ being global dlopen.
Now we do have an option to change this and yes, default that to local.

This is also the reason that we need a NEWS entry to make the users aware of "previously system specific, now we ask the dynamic link loader to ...".